### ISSUE TITLE
Content deleted when formatting with indent_handlebars: true

### ISSUE THREAD

**Issue by aeschli** [beautify-web/js-beautify#1174]
Format the following html code with setting `indent_handlebars: true`

# Input

```
<h3>
{{outlet}}
{{!--{{log 'a'}}--}}
</h3>
```

# Actual Output
The  code looks like this after beautification:
```
<h3>
    {{!--{{log 'a'}}--}}
</h3>
```

# Expected Output
The code should have looked like this after beautification:
```
<h3>
    {{outlet}}
    {{!--{{log 'a'}}--}}
</h3>
```

**Comment by beastmst** [comment]
when can this bug can be fixed?

**Comment by MacKLess** [comment]
This appears to have been fixed as of v1.8.0-rc14. I am writing tests and will be submitting a PR by the end of the day.
